src/client/src/App/Trades/TradesGrid/NumFilter.tsx

```diff
@@ -69,6 +69,7 @@ export const NumFilter = ({
     <FilterPopup
       parentRef={parentRef}
       ariaLabel={`Filter trades by ${colDef[field].headerName} field value`}
+      leftAlignFilter={field !== "tradeId"}
```

Comment:
The alignment for numeric field in the blotter were not displayed in the screen. Please see [ReactiveTrader](https://www.reactivetrader.com/) and click the Filter Icon for Notional, and Rate. This is currently fixed, see image below:
 
![numeric_dialog](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/assets/10324761/adf1f377-b9a6-4f27-97f8-69f1a20b72f3)